tests/components/sensor/test_yr.py

```diff
@@ -32,12 +31,9 @@ def test_default_setup(self, betamax_session):
                    return_value=betamax_session):
             with patch('homeassistant.components.sensor.yr.dt_util.utcnow',
                        return_value=now):
-                assert sensor.setup(self.hass, {
-                    'sensor': {
-                        'platform': 'yr',
-                        'elevation': 0,
-                    }
-                })
+                    assert _setup_component(self.hass, 'sensor', {
```

Comment:
Since each test in this file will receive a fresh Home Assistant instance, we do not have to reset the loaded components.